sky/cli.py

```diff
@@ -3736,6 +3737,7 @@ def spot_dashboard(port: Optional[int]):
 
     with subprocess.Popen(ssh_command, shell=True,
                           start_new_session=True) as ssh_process:
+        time.sleep(5)  # Added delay for ssh_command to initialize.
         webbrowser.open(f'http://localhost:{free_port}')
```

Comment:
BTW, I realized a mistake - L3722 `or use `sky start` to bring up an existing controller.'` should be removed. Simply running `sky start` on a stopped controller does not auto-restart the dashboard.

Reply:
Removed